static/scss/answers/templates/vertical-standard.scss

```diff
@@ -23,6 +30,7 @@
 
   &-filtersWrapper {
     margin-bottom: 20px;
+    float: right;
```

Comment:
what are your thoughts on using flexbox for this as well? I think justify-content flex-end might work

Reply:
hmm, I used float since it only deal with positioning a small section without much big layout changes. But instead of just doing this for the filter, I think it will be better to have the left container use flexbox and justify-content flex-end for more consistent scss layout. Updated in the new commit. Thanks for the suggestion!